Jenkinsfile

```diff
@@ -30,10 +30,12 @@ pipeline{
 		stage('Setup: Download Ortholog files from PANTHER'){
 			steps{
 				script{
-					sh "wget -q ftp://ftp.pantherdb.org/ortholog/current_release/Orthologs_HCOP.tar.gz"
-					sh "tar -xvf Orthologs_HCOP.tar.gz"
-					sh "wget -q ftp://ftp.pantherdb.org/ortholog/current_release/QfO_Genome_Orthologs.tar.gz"
-					sh "tar -xvf QfO_Genome_Orthologs.tar.gz"
+					def hcopFilename = "Orthologs_HCOP.tar.gz";
+					def qfoFilename = "QfO_Genome_Orthologs.tar.gz";
+					sh "wget -q ftp://ftp.pantherdb.org/ortholog/current_release/${hcopFilename}"
+					sh "tar -xvf ${hcopFilename}"
+					sh "wget -q ftp://ftp.pantherdb.org/ortholog/current_release/${qfoFilename}"
+					sh "tar -xvf ${qfoFilename}"
```

Comment:
Nice - the variable names are good.  You could add one for the Panther Current Release URL "ftp://ftp.pantherdb.org/ortholog/current_release"